### PR TITLE
OncoKB : Remove the navCaseIds for the feedback

### PR DESCRIPTION
**Pull request by zhx828**
Fix https://github.com/cBioPortal/cbioportal/issues/3754

### CODE REVIEW

File: src/shared/components/annotation/OncoKB.tsx [hunk 1]
```diff
@@ -170,7 +170,7 @@ export default class OncoKB extends React.Component<IOncoKbProps, {}>
         const geneParam = `entry.1744186665=${hugoSymbol || ''}`;
         const alterationParam = `entry.1671960263=${alteration || ''}`;
         const userParam = `entry.1381123986=${this.props.userEmailAddress || ''}`;
-        const uriParam = `entry.1083850662=${encodeURIComponent(window.location.href)}`;
+        const uriParam = `entry.1083850662=${encodeURIComponent(window.location.href.replace(/navCaseIds=[^&]*/, ''))}`;
```
Comment (alisman) [inline src/shared/components/annotation/OncoKB.tsx]: probably better to just build window.location.hostname+window.location.path+window.location.search. but i'll approve and you make change if you want

Comment (zhx828) [reply src/shared/components/annotation/OncoKB.tsx]: @alisman I'm thinking to keep other parameters after the pound sign

Comment (alisman) [reply src/shared/components/annotation/OncoKB.tsx]: i just don't want to have that parameter hardcoded.  i don't think the hash will be important

Comment (zhx828) [reply src/shared/components/annotation/OncoKB.tsx]: Thanks. Updated the url.

**Pull request merged**